pages/common/kubectl.md

```diff
@@ -34,3 +26,11 @@
 - Display an explanation of the specific field:
 
 `kubectl explain pods.spec.containers`
+
+- Print the logs for a container in a pod or specified resource:
+
+`kubectl logs my-pod`
```

Comment:
I would suggest changing `my-pod` to `pod_name` just to follow the general convention in tldr-pages.

Reply:
I would also suggest to wrap it: `{{pod_name}}`.

Reply:
Agree ^